.travis.yml

```diff
@@ -1,13 +1,34 @@
 language: go
-env:
-  - GO111MODULE=on
+
+matrix:
+  include:
+    - os: linux
+      go: "1.11.x"
+      env: GO111MODULE=on CHECK_GOFMT=1
+    - os: osx
+      go: "1.11.x"
+      env: GO111MODULE=on
+    - os: windows
+      go: "1.11.x"
+      env: GO111MODULE=on
```

Comment:
FYI, you could just do:

```
env:
  global:
    - GO111MODULE=on
```

Then any `env` lines in the matrix include will be appended. I do that here: https://github.com/mvdan/sh/blob/master.v3/.travis.yml

Reply:
Ah, thanks. That was the thing I was missing, the `global` bit. Will fix.